README.md

```diff
@@ -607,25 +533,25 @@ The official OCS samples are divided in multiple categories depending on the sce
   </tr>
   <tr>
     <th align="middle" colspan="2">
-      <h2>Analytics</h2>
+      <h2><a href="https://github.com/osisoft/OSI-Samples-OCS/docs/ANALYTICS.md">Analytics</a></h2>
     </th>
   </tr>
   <tr>
     <td align="middle" valign="top">
       <b>
-        <a href="docs/DATA_VIEWS_README.md"> Data Views </a>
+        <a href="docs/DATA_VIEWS.md"> Data Views </a>
       </b>
       <br />
       These samples highlight basic operations of Data Views for OCS, including
       creation, updating, getting data from and deletion of Data Views.
-      <a href="docs/DATA_VIEWS_README.md">Details</a>
+      <a href="docs/DATA_VIEWS.md">Details</a>
       <br />
       <br />
       <table align="middle">
         <tr>
           <td align="middle">
             <a href="https://github.com/osisoft/sample-ocs-data_views-java">
-              Java
+              JAVA
```

Comment:
Why did we capitalize `JAVA` everywhere? I don't see it in all caps on the web anywhere..

Reply:
mainly because it is fun, but we can go to Java